templates/editStages.phtml

```diff
@@ -7,9 +7,12 @@
     echo \Portal\ViewHelpers\LayoutViewHelper::outputTitle('Edit Applicant Stages');
     ?>
     <link href="css/stages.css" rel="stylesheet" type="text/css">
+    <link href="css/stageDeletionModal.css" rel="stylesheet" type="text/css">
     <script defer src="./js/Sortable.js"></script>
     <script defer src="./js/stagesPage.js"></script>
     <script defer src="./js/stageReorder.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.15.4/js/all.js" data-auto-a11y="true" ></script>
```

Comment:
wanna move this up the top above the other links so it can be overridden with your js